src/AppInstallerCLITests/WorkFlow.cpp

```diff
@@ -782,6 +815,30 @@ TEST_CASE("ShowFlow_SearchAndShowAppVersion", "[ShowFlow][workflow]")
     REQUIRE(showOutput.str().find("  Download Url: https://ThisIsNotUsed") == std::string::npos);
 }
 
+TEST_CASE("ShowFlow_ShowDependencies", "[ShowFlow][workflow][showDependencies]")
+{
+    std::ostringstream showOutput;
+    TestContext context{ showOutput, std::cin };
+    context.Args.AddArg(Execution::Args::Type::Manifest, TestDataFile("Manifest-Good-AllDependencyTypes.yaml").GetPath().u8string());
+
+    TestUserSettings settings;
+    settings.Set<AppInstaller::Settings::Setting::EFExperimentalShowDependencies>({true});
+
+    ShowCommand show({});
+    show.Execute(context);
+    INFO(showOutput.str());
+
+    // Verify all types of dependencies are printed
+    REQUIRE(showOutput.str().find("Dependencies") != std::string::npos);
+    REQUIRE(showOutput.str().find("WindowsFeaturesDep") != std::string::npos);
+    REQUIRE(showOutput.str().find("WindowsLibrariesDep") != std::string::npos);
+    // PackageDep1 has minimum version (1.0), PackageDep2 doesn't (shouldn't show [>=...])
+    REQUIRE(showOutput.str().find("Package.Dep1-x64\x1b[0m [>= 1.0]") != std::string::npos);

```

Comment:
I think we should explicitly be looking not to output these extraneous characters.  I'm going to guess that they are being output as the "reset" state for the Info stream.  Instead of using `Info()` repeatedly, simply capturing the object for reuse prevents that:
```
auto info = context.Info();
```